OasisDEX/oasis-borrow
# Fix endless loading spinner bug on My Positions view

BugFix. Endless loading spinner on My Positions view when account has no open positions.

  ## Changes 👷‍♀️
- update vaultWithAutomationData$ pipe to ensure it returns an observable that emits an empty array when no vaults are found

Note: This change ensures that users with no open positions will see the no-open vaults variant of the My positions - and won't simply see an endless loading spinner.
  
## How to test 🧪
- Start a hardhat node, 
- navigate to My Positions with an account that has no open positions - confirm that the loading spinner does not spin indefinitely
- Then, setup a position and once the position is created, navigate back to My Positions and confirm the position is visible as expected
